### PR TITLE
dropbox to output the code as a polaroid photo or a regular sheet

### PR DESCRIPTION
**Pull request by Evan4ik**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,14 +69,23 @@
...
         } catch(e) {
             output.value = `Error while converting! ${e}\n\nDoes your song span too many octaves?`;
+            oT += "errorLOL"
         }
         e.preventDefault();
+        if (oT == "photo") {
+            photoCode = '[["PolaroidPhoto",[],{"Phrase":"' + convert(map, input.value.split('')).join('') + '"}]]';
+            photoCode = btoa(photoCode)
+            output.value = photoCode;
+        }
```
Comment (MightyAlex200) [inline index.html]: Everything else seems fine but this looks very sloppy. Could you rework this to be less destructive to the original logic of the applet? If not I would be willing to do it.

File: index.html [hunk 2]
```diff
@@ -137,10 +146,18 @@ <h1>Gramby's Auditorium</h1>
             <input type="submit" value="Convert"> 
         </form>
         <div>
+            <label for="outputType">Output type:</label>
+                <select name="outputType" id="outputType">
```
Comment (MightyAlex200) [inline index.html]:
```suggestion
            <select name="outputType" id="outputType">
```

**Comment by Evan4ik** [comment]
bip

**Review by MightyAlex200 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Evan4ik) [reply index.html]: i'll try to clean it up

Comment (Evan4ik) [reply index.html]: fixed it a bit, it looks a little better now. please point out what I could try to fix some more.

**Review by MightyAlex200 — APPROVED** [review]

**Pull request merged**